core/templates/dev/head/pages/topics_and_skills_dashboard/skills_list_directive.html

```diff
@@ -0,0 +1,84 @@
+<div class="skills-list">
+  <md-card layout="row" class="list-view-item">
+    <table class="dashboard-table">
+        <colgroup>
```

Comment:
This and the next four lines: deindent by 2

Reply:
Done.